src/server/req_manager.c

```diff
@@ -1686,6 +1692,13 @@ mgr_queue_set(struct batch_request *preq)
 		rc = mgr_set_attr(pque->qu_attr, que_attr_def, QA_ATR_LAST,
 			plist, preq->rq_perm, &bad, (void *)pque,
 			ATR_ACTION_ALTER);
+		if (rc == PBSE_CANNOT_SET_ROUTE_QUE) {
```

Comment:
The error code PBSE_CANNOT_SET_ROUTE_QUE is being set in set_queue_type() action function only. We thought of logging this in server as well if this error occurs. 

Also when I do not have this block of code here, it was actually appending the name of the attribute to the error message which is queue_type something like 
"Unable to set queue_type to route queue with the partition attribute set **queue_type**"
But the requirement is to exactly get the following message where queue_type should not be appended.
"Unable to set queue_type to route queue with the partition attribute set"